CMakeLists.txt

```diff
@@ -215,6 +215,7 @@ if(BUILD_TOOLS AND WIN32)
     Meshconvert/Mesh.h
     Meshconvert/Mesh.cpp
     Meshconvert/SDKMesh.h)
+  target_link_options(meshconvert PRIVATE -municode)
```

Comment:
For MinGW this is already set on line 300. If this is specific to the clang complier, it should be guarded as such down around 295.

Reply:
I compile for the target mingw-w64 using BOTH Clang and GCC.  The build works on GCC but does not work on Clang. with the following error:

```
[22/22] Linking CXX executable bin\CMake\meshconvert.exe
FAILED: bin/CMake/meshconvert.exe
C:\WINDOWS\system32\cmd.exe /C "cd . && C:\msys64\clang64\bin\c++.exe   CMakeFiles/meshconvert.dir/Meshconvert/Meshconvert.cpp.obj CMakeFiles/meshconvert.dir/Meshconvert/meshconvert.rc.obj CMakeFiles/meshconvert.dir/Meshconvert/MeshOBJ.cpp.obj CMakeFiles/meshconvert.dir/Meshconvert/Mesh.cpp.obj -o bin\CMake\meshconvert.exe -Wl,--out-implib,bin\CMake\libmeshconvert.dll.a -Wl,--major-image-version,0,--minor-image-version,0  bin/CMake/libDirectXMesh.a  -lversion  C:/msys64/clang64/lib/libDirectX-Headers.a  -lkernel32 -luser32 -lgdi32 -lwinspool -lshell32 -lole32 -loleaut32 -luuid -lcomdlg32 -ladvapi32 && cd ."
ld.lld: error: undefined symbol: WinMain
>>> referenced by C:/M/B/src/mingw-w64/mingw-w64-crt/crt/crtexewin.c:67
>>>               libmingw32.a(lib64_libmingw32_a-crtexewin.o):(main)
c++: error: linker command failed with exit code 1 (use -v to see invocation)
ninja: build stopped: subcommand failed.

```
I'll  have to reevaluate given what you pointed out to me because I know why the build always worked on GCC/